### ISSUE TITLE
landscape view shows very little info, could use condensing

### ISSUE THREAD

**Issue by nutpantz** [davidtakac/prognoza#67]
![Screenshot_20231005_234303_Prognoza](https://github.com/davidtakac/prognoza/assets/12160507/2287def2-717c-48fe-8b9b-323e46c63a6c)

Landscape mode shows very little data without scrolling.

You could get so much more info on the screen with a little condensing for landscape
I suspect you could fit hourly and maybe a 3 day forecast in that view.

Otherwise the app has much better information presentation then most others

**Comment by davidtakac** [comment]
Indeed. I was thinking of addressing this in the redesign (issue #32) by splitting the horizontal screen into two chunks. The left could have Now, Hourly and Daily forecast, and the right will have the detail tiles.
